src/patch_parser.py

```diff
@@ -390,22 +429,34 @@ def _parse_patch_file_unchanged_chunk(
                                   side='',
                                   offset=(gerrit_new_line - lines.line_number())))
 
+
 def _parse_patch_file_added_chunk(
         lines: InputSource,
         gerrit_orig_line: int,
         gerrit_new_line: int) -> Tuple[int, int, PatchFileChunkLineMap]:
     in_start = lines.line_number()
     logging.info('First char - 1: %c', lines[0][0])
     while lines[0] and lines[0][0] == '+':
+        previous = lines.get_previous_line()
+        print('ooc previous', previous)
+        print('ooc Item 1 -> ', lines[0])
```

Comment:
These look like debugging lines. If they are, remove them. If not, please log them.